app/styles/layouts/auth.css

```diff
@@ -17,6 +17,10 @@
     margin-bottom: 1.5rem;
 }
 
+.gh-signin .form-group.error {
```

Comment:
@kevinansfield No, the behavior hasn't changed at all, but I noticed that any error message text between the fields of the signin form is squished between them. I'm fairly certain this exists on master too, but will double check.